ftw/contacts/configure.zcml

```diff
@@ -22,6 +22,7 @@
 
     <include package=".geo" zcml:condition="have ftw.contacts:geo" />
     <include package=".simplelayout" zcml:condition="have ftw.contacts:simplelayout" />
+    <include package=".sync"  zcml:condition="installed ldap" />
```

Comment:
Please check for the extras instead of a single dependency; see [ftw.autofeature](https://github.com/4teamwork/ftw.autofeature) (already configured here).
```zcml:condition="have ftw.contacts:ldap"```